hdf5/cppbuild.sh

```diff
@@ -19,6 +19,15 @@ echo "Decompressing archives..."
 tar --totals -xf ../hdf5-$HDF5_VERSION.tar.bz2
 cd hdf5-$HDF5_VERSION
 
+# Copy SIS JHDF5 helper files
+# Copyright 2007 - 2018 ETH Zuerich, CISD and SIS
+# Apache License, Version 2.0
+# https://sissource.ethz.ch/sispub/jhdf5
+cp ../../../src/main/c/jhdf5/*.c java/src/jni/
```

Comment:
Please don't add them to this repository. Download them just like HDF5.

Reply:
The files from JHDF5 need significant modification since they were last used with HDF 1.10.

I could start a branch on my fork and pull from there. Would that be better?

https://github.com/JaneliaSciComp/jhdf5

In the meantime, could we run the CI so I can see if there are any cross platform issues?

Reply:
Why not just rewrite it in Java then?

Reply:
I think that might be the easiest route. I'll revert recent changes